### DISCUSSION TITLE
How to access a proxy from another proxy?

### DISCUSSION THREAD

**Discussion by Kilowhisky** [pmndrs/valtio#983] [Q&A]
Got probably a simple question that i'm overcomplicating with the wrong structure. 

I am storing the users session as a Valtio proxy, this contains the user's id.  
```
export interface SessionStore {
  isLoading: boolean,
  isLoaded: boolean,
  isAuthenticated: boolean,
  session: Session | null
}

export const sessionStore = proxy<SessionStore>({
  isLoading: false,
  isLoaded: false,
  isAuthenticated: false,
  session: null
});
```

I need to get the user's id in most subsequent calls in the other proxies. How can i do that?

For example, here is the users ProfileStore.  In order to load the user's profile, I need the users identifier which is in another proxy. 

Is there a way to do this without having to utilize a `load(user_id)` action?

```
  export type ProfileStore = {
    tableData: Promise<Database['public']['Tables']['user_profiles']['Row']>
  }

  export const profileStore = proxy<ProfileStore>({
    tableData: assurePromise(supabase
      .from('user_profiles')
      .select()
      .eq('id', /* USER ID FROM SESSION STORE */)
      .limit(1)
      .single()
      .then()
    )
  });
```

**Comment by dai-shi** [comment]
What would you do if it's not a `proxy()`? It must be the same thing.
